src/lib/components/modal/DuplicateSource.svelte

```diff
@@ -0,0 +1,45 @@
+<script lang="ts">
+  import {
+    DuplicateActions,
+    duplicateSourceAction,
+    duplicateSourceName,
+  } from "$lib/application-state-stores/application-store";
+  import Modal from "$lib/components/modal/Modal.svelte";
+  import ModalAction from "$lib/components/modal/ModalAction.svelte";
+  import ModalActions from "$lib/components/modal/ModalActions.svelte";
+  import ModalContent from "$lib/components/modal/ModalContent.svelte";
+  import ModalTitle from "$lib/components/modal/ModalTitle.svelte";
+</script>
+
+<Modal open={$duplicateSourceName !== null} onBackdropClick={() => undefined}>
+  <ModalTitle>duplicate source found</ModalTitle>
+  <ModalContent
+    >a source with the name {$duplicateSourceName} already exists</ModalContent
```

Comment:
Capitalise the 1st char in this model.

Reply:
Have made it lowercase based on previous conversation with @ericpgreen2 (https://github.com/rilldata/rill-developer/pull/438#discussion_r907765232)

All text in modals, overlays etc. are lowecase

Reply:
Changed case after discussion with @magorlick